pages/about/contributing-in-github.md

```diff
@@ -0,0 +1,119 @@
+
+4. Our chapters contain metadata, please fill it out.
+
+![](/assets/img/1mQ_Image_1.png)
+
+**Tip:** For more information on writing and formatting, visit [GitHub's documentation](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github).
+
+## **Start editing**
+
+1. Go to the [DS Handbook](https://github.com/elixir-uk/elixir-ds-handbook) on GitHub.
+
+2. Click **Issues** in the top menu bar or use this [link](https://github.com/elixir-uk/elixir-ds-handbook/issues).
+
+- **If a similar issue exists**, add a comment in the issue to share your suggestions.
+
+- **If no relevant issue exists**, create a new one. you can find information on creating issues in the [GitHub documentation](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue).
+
+![](/assets/img/yWr_Image_2.png)
+
+- <!-- Discuss your idea with the editors through comments in the issues ? you will be notified when others comment.
+
+- You can read the comments and write your opinion/questions/answers in the **Leave a comment** box. To submit your responses, click the green **Comment** button on the right.
+
+- You can always return to your opened issue by going to the[issues section](https://github.com/elixir-europe/rdmkit/issues) of our GitHub repo.  \
+-->
+
+- Find where to edit on GitHub
+
+- If you want to contribute to an existing chapter, go to the chapter on the site and click the **Edit me** pencil icon next to the page title.
+
+![](/assets/img/6U1_Image_3.png)
+
+![](/assets/img/hUE_Image_4.png)
+
+- Copy content of a template into new page.
+
+- You will be taken to the correct GitHub repository, where you will look for a pencil icon on the top right.
+
+![](/assets/img/oCM_Image_5.png)
+
+<!--
+
+## **Linking resources and other pages (optional)**
+
+- If you have mentioned tools or resources in your text, you will have to add them to the[ ](https://rdmkit.elixir-europe.org/tool_resource_update)[tool and resource list](https://rdmkit.elixir-europe.org/tool_resource_update).
+
+- If you want to list training material or link to other RDMkit pages, add it to the page metadata. Read more on how to do this in our[ ](https://rdmkit.elixir-europe.org/page_metadata)[page metadata guide](https://rdmkit.elixir-europe.org/page_metadata).
+
+**Important:** In general terms, you must avoid manual interlinking of RDMkit pages.
+
+-->
+
+## **Submit your changes**
+
+1. When you are done with your changes, commit them by clicking the green button at the top right corner.
+
+![](/assets/img/UTC_Image_6.png)
+
+2. This directs you to the **Propose changes** section where you need to write a title and a brief explanation of your changes.
+
+![](/assets/img/fH5_Image_7.png)
+
+3. Click **Propose changes**.
+
+![PrDraft pull request on GitHub](/assets/img/Gfi-prdraft-pull-request-github.png)
+
+![Linking issues in a pull request on GitHub](/assets/img/CI7-linking-issues-pull-request-github.png)
```

Comment:
Now that abbreviation is defined (see previous comment), from here onwards you can simply use the word PR instead of pull request.

Reply:
True, although I think we can leave it spelled out as it is a new concept for the target audience so worth repeating in the text.